source/about/self-hosted-subscriptions.rst

```diff
@@ -125,26 +125,26 @@ We'll send you an email notice around the end of the quarter reminding you to se
 
 To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. 
 
-- Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric. 
+- Please ensure your screenshot is taken from the top of the page and includes the **Total Activated Users** metric. 
```

Comment:
I wonder how we communicate the fact that for versions before v9.5, in the System Console it'd still be "Total Active Users"

Reply:
Good catch. I've updated the bullet point to clarify.